src/components/Scheme/SchemeContainer.tsx

```diff
@@ -30,13 +32,18 @@ interface IExternalProps extends RouteComponentProps<any> {
   daoState: IDAOState;
 }
 
-interface IStateProps {
+interface IExternalStateProps {
```

Comment:
what does ExternalStateProps mean?

Reply:
State that is coming from outside the component (in this case, `match.params.schemeId`)  (see `mapStateToProps`).  To be consistent with your request below, I'm renaming to `IExternalState`.

Reply:
these are not state items though they are props coming from the state (mapStateToProps) that is why it is called IStateProps

Reply:
these are not state items though they are props coming from the state (mapStateToProps) that is why it is called IStateProps

Reply:
I just like to distinguish between the internal state and the external state.  `IExternalState` does end up being part of `IProps`, so seems like `IExternalStateProps` makes the most sense.  Does that make sense to you?

Reply:
what makes this one External though? as opposed to everywhere where we use IStateProps

Reply:
So I guess 'IExternalStateProps' is inconsistent, but `IStateProps` makes me think it is part of the component's state, which it isn't.  Maybe I have to get used to `IState` just meaning the component's state, and `IStateProps` meaning props that come from the external (not internal) state.

Reply:
So I'm renaming to `IStateProps`

Reply:
yeah, props are things that come from somewhere else (external), state is the internal state. IStateProps comes from the Redux global state, so maybe we should rename IStateProps to IReduxStateProps. actually I guess IExternalStateProps is ok by me... if you want to rename all of them go for it.

Reply:
LOL, maybe another time  :-)